components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -786,82 +786,100 @@ IJsonSerializer jsonSerializer
             CancellationToken ct) {
             _logger.Information("{nameof} method triggered", nameof(UnpublishAllNodesAsync));
             var sw = Stopwatch.StartNew();
+            // when no endpoint is specified remove all the configuration
+            var purge = null == request.EndpointUrl;
```

Comment:
Let's document that this behavior is coming from `2.5.x` and we have it here for backward-compatibility.

Reply:
Note about backward compatibility added in #1596.